### PR TITLE
feature 1 done with its all of its methods and controller

### PR DESCRIPTION
**Pull request by AlejandroQuiceno**
wenas

### CODE REVIEW

**Comment by Rivercoder201** [comment]
you didnt do all what we asked you to do, so do all the methods that are in the hack and plan

**Review by Rivercoder201 — CHANGES_REQUESTED** [review]
you need to have beter sintaxis in your code, its garbage

**Comment by AlejandroQuiceno** [comment]
ok

File: Feature 1.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Hola buenas estos son los diferentes metodos que vamos a usar en nuestro proyecto
+
+metodo{
```
Comment (Rivercoder201) [inline Feature 1.txt:3]: la verdad aqui no se que putas hiciste

Comment (Rivercoder201) [reply Feature 1.txt:3]: pero bueno eres re pendejp

**Review by Rivercoder201 — CHANGES_REQUESTED** [review]

**Review by AlejandroQuiceno — COMMENTED** [review]
listo ya estan hechos

**Review by Rivercoder201 — APPROVED** [review]

**Pull request merged**